Broken production build on 9b254b2ee

Current tip of develop (`9b254b2ee`) throws an error while compiling with `--no-default-features` flag:
```
   Compiling holochain v0.0.100 (/Users/pj/Programming_School/holochain/holochain/holochain/crates/holochain)
error[E0433]: failed to resolve: use of undeclared crate or module `kitsune_p2p_types`
  --> crates/holochain/src/conductor/handle.rs:67:5
   |
67 | use kitsune_p2p_types::config::JOIN_NETWORK_TIMEOUT;
   |     ^^^^^^^^^^^^^^^^^ use of undeclared crate or module `kitsune_p2p_types`

error[E0425]: cannot find value `JOIN_NETWORK_TIMEOUT` in this scope
   --> crates/holochain/src/conductor/handle.rs:710:44
    |
710 |                 match tokio::time::timeout(JOIN_NETWORK_TIMEOUT, network.join()).await {
    | 
```

--- a/crates/holochain/Cargo.toml
+++ b/crates/holochain/Cargo.toml
@@ -37,6 +37,7 @@ holochain_wasmer_host = "=0.0.67"
 holochain_websocket = { version = "0.0.1", path = "../holochain_websocket" }
 holochain_zome_types = { version = "^0.0.2-alpha.1", path = "../holochain_zome_types" }
 human-panic = "1.0.3"
+kitsune_p2p_types = { version = "0.0.1", path = "../kitsune_p2p/types" }
 lazy_static = "1.4.0"
 mockall = "0.8"
 mr_bundle = { version = "0.0.1", path = "../mr_bundle" }
@@ -77,7 +78,6 @@ matches = {version = "0.1.8", optional = true }
 holochain_test_wasm_common = { version = "0.0.1", path = "../test_utils/wasm_common", optional = true  }
 unwrap_to = { version = "0.1.0", optional = true }
 itertools = { version = "0.10", optional = true }
-kitsune_p2p_types = { version = "0.0.1", path = "../kitsune_p2p/types" }
 
 
 [dev-dependencies]
@@ -119,6 +119,7 @@ test_utils = [
   "holochain_state/test_utils",
   "holochain_types/test_utils",
   "holochain_zome_types/test_utils",
+  "kitsune_p2p_types/test_utils",
   "itertools",
   "matches",
   "holochain_test_wasm_common",

--- a/crates/holochain/Cargo.toml
+++ b/crates/holochain/Cargo.toml
@@ -77,7 +77,7 @@ matches = {version = "0.1.8", optional = true }
 holochain_test_wasm_common = { version = "0.0.1", path = "../test_utils/wasm_common", optional = true  }
 unwrap_to = { version = "0.1.0", optional = true }
 itertools = { version = "0.10", optional = true }
-kitsune_p2p_types = { version = "0.0.1", path = "../kitsune_p2p/types", optional = true }
+kitsune_p2p_types = { version = "0.0.1", path = "../kitsune_p2p/types" }
 
 
 [dev-dependencies]
@@ -119,8 +119,6 @@ test_utils = [
   "holochain_state/test_utils",
   "holochain_types/test_utils",
   "holochain_zome_types/test_utils",
-  "kitsune_p2p_types",
-  "kitsune_p2p_types/test_utils",
   "itertools",
   "matches",
   "holochain_test_wasm_common",

